crates/symbolicator-service/src/config.rs

```diff
@@ -293,15 +293,28 @@ pub struct InMemoryCacheConfig {
     ///
     /// Defaults to `100`.
     pub s3_client_capacity: u64,
+
+    /// Capacity (in bytes) for the in-memory `object_meta` Cache.
+    ///
+    /// Defaults to `100 MiB (= 104_857_600)`.
+    pub object_meta_capacity: u64,
+
+    /// Capacity (in bytes) for the in-memory `cficaches` Cache.
+    ///
+    /// Defaults to `400 MiB (= 419_430_400)`.
+    pub cficaches_capacity: u64,
```

Comment:
To be honest I'm not too fond of the entire `InMemoryCacheConfig` struct. For these options in particular I think it would be nicer if they lived directly in their respective cache configurations (i.e. add an `in_memory_capacity` field to `DownloadedCacheConfig` and `DerivedCacheConfig`), but I understand that would make setting these defaults impossible.

Reply:
I would rather like to get rid of the difference between `DownloadedCacheConfig` and `DerivedCacheConfig` altogether. The comment I posted above shows that we have bugs related to the `retry_after` timings. And the only difference between the two is the time-to-idle, which we could express as their own explicit config settings.

Reply:
Yeah actually that would be even better.